planner/optimizer/optimizer-core/src/test/resources/log4j.xml

```diff
@@ -4,15 +4,51 @@
 	xmlns:log4j='http://jakarta.apache.org/log4j/'>
 
 	<appender name="console" class="org.apache.log4j.ConsoleAppender">
```

Comment:
do you think it is possible to use something like https://wiki.apache.org/logging-log4j/FileAppender ?
Otherwise happy to merge it as-is.